test/apps/iodemo/io_demo.cc

```diff
@@ -88,6 +88,10 @@ class MemoryPool {
         if (_freelist.empty()) {
             item = new T(_buffer_size, *this);
             _num_allocated++;
+            if (_verbose && ((_num_allocated % 100) == 0)) {
+                UcxLog("MEM_POOL", _verbose) << "pool " << this
+                                             << ": allocated " << _num_allocated;
+            }
```

Comment:
print this along with printing bandwidth

Reply:
but then we can control only data buffers pool, this print allows to catch any pool leak

Reply:
print all pools status in same line where bw is printed:
"data-pool: %u/%u cb-pool: %u/%u", inuse/total-allocated, inuse/total-allocated

Reply:
ok, but only client prints bandwidth

Reply:
good point
let's move the print to next PR to polish it later

Reply:
in addition, we report perf when window is done, so inuse == 0

Reply:
reverted